### PR TITLE
Fix a bug in the computation of the inverse of a rigid motion

### PR DESCRIPTION
**Pull request by eggrobin**
And switch to saner conventions.

### CODE REVIEW

**Comment by eggrobin** [comment]
The actual bugfix is https://github.com/eggrobin/Principia/commit/0151a993b4fad54c14720984c3db45040765bcb8.

File: physics/body_centered_non_rotating_dynamic_frame_body.hpp [hunk 1]
```diff
@@ -46,9 +46,9 @@ FromThisFrameAtTime(Instant const& t) const {
                            centre_degrees_of_freedom.position(),
                            Identity<ThisFrame, InertialFrame>().Forget());
   return RigidMotion<ThisFrame, InertialFrame>(
-             rigid_transformation,
-             AngularVelocity<InertialFrame>(),
-             centre_degrees_of_freedom.velocity());
+      rigid_transformation, AngularVelocity<ThisFrame>(),
```
Comment (pleroy) [inline physics/body_centered_non_rotating_dynamic_frame_body.hpp]: One parameter per line.

File: physics/rigid_motion_body.hpp [hunk 2]
```diff
@@ -13,20 +13,12 @@ namespace physics {
...
-template<typename FromFrame, typename ToFrame>
-RigidMotion<FromFrame, ToFrame>::RigidMotion(
-    RigidTransformation<FromFrame, ToFrame> const& rigid_transformation,
-    AngularVelocity<ToFrame> const& rotation,
-    Velocity<ToFrame> const& translation)
-    : rigid_transformation_(rigid_transformation),
-      rotation_(rotation),
-      translation_(translation) {}
+      angular_velocity_of_to_frame_(
+          angular_velocity_of_to_frame),
```
Comment (pleroy) [inline physics/rigid_motion_body.hpp]: May fit on the previous line.

File: physics/rigid_motion_body.hpp [hunk 3]
```diff
@@ -62,8 +59,10 @@ RigidMotion<FromFrame, ToFrame> operator*(
     RigidMotion<FromFrame, ThroughFrame> const& right) {
   return RigidMotion<FromFrame, ToFrame>(
       left.rigid_transformation() * right.rigid_transformation(),
-      left.rotation_ + left.orthogonal_map()(right.rotation_),
-      left(right({FromFrame::origin, Velocity<FromFrame>()})).velocity());
+      right.angular_velocity_of_to_frame_ +
+          right.orthogonal_map().Inverse()(left.angular_velocity_of_to_frame_),
+      right.Inverse()(left.Inverse()({ToFrame::origin, Velocity<ToFrame>()}))
```
Comment (pleroy) [inline physics/rigid_motion_body.hpp]: Preference for splitting before {

File: physics/rigid_motion_test.cpp [hunk 4]
```diff
@@ -132,8 +139,55 @@ TEST_F(RigidMotionTest, ApparentMoon) {
       Wedge(earth_to_moon, moon_degrees_of_freedom.velocity()) /
       InnerProduct(earth_to_moon, earth_to_moon) * Radian;
   EXPECT_THAT(moon_angular_velocity / (2 * π * Radian) * Day,
-              Componentwise(0, 0, AlmostEquals(-29.0 / 30.0, 0)));
+              Componentwise(0, 0, AlmostEquals(-29.0 / 30.0, 5)));
 }
 
+TEST_F(RigidMotionTest, GroupoidAssociativity) {
+  auto terrestrial_to_geocentric = geocentric_to_terrestrial_.Inverse();
```
Comment (pleroy) [inline physics/rigid_motion_test.cpp]: auto const

File: physics/rigid_motion_test.cpp [hunk 5]
```diff
@@ -132,8 +139,55 @@ TEST_F(RigidMotionTest, ApparentMoon) {
...
+      (selenocentric_to_lunar_ *
+       (geocentric_to_selenocentric_ * terrestrial_to_geocentric))(
+          degrees_of_freedom_);
+  EXPECT_THAT(d1.position() - Lunar::origin,
+              AlmostEquals(d2.position() - Lunar::origin, 0));
+  EXPECT_THAT(d1.velocity(), AlmostEquals(d2.velocity(), 4));
+}
+
+TEST_F(RigidMotionTest, GroupoidAction) {
+  auto terrestrial_to_geocentric = geocentric_to_terrestrial_.Inverse();
```
Comment (pleroy) [inline physics/rigid_motion_test.cpp]: Same (2x).

File: physics/rigid_motion_test.cpp [hunk 6]
```diff
@@ -132,8 +139,55 @@ TEST_F(RigidMotionTest, ApparentMoon) {
...
+      (geocentric_to_lunar * terrestrial_to_geocentric)(degrees_of_freedom_);
+  DegreesOfFreedom<Lunar> const d2 =
+      geocentric_to_lunar(terrestrial_to_geocentric(degrees_of_freedom_));
+  EXPECT_THAT(d1.position() - Lunar::origin,
+              AlmostEquals(d2.position() - Lunar::origin, 5));
+  EXPECT_THAT(d1.velocity(), AlmostEquals(d2.velocity(), 1));
+}
+
+TEST_F(RigidMotionTest, GroupoidInverse) {
+  auto terrestrial_to_lunar = selenocentric_to_lunar_ *
```
Comment (pleroy) [inline physics/rigid_motion_test.cpp]: Same.

File: physics/body_centered_non_rotating_dynamic_frame_body.hpp (hunk 1 above)
Comment (eggrobin) [inline physics/body_centered_non_rotating_dynamic_frame_body.hpp]: Done.

File: physics/rigid_motion_body.hpp (hunk 2 above)
Comment (eggrobin) [inline physics/rigid_motion_body.hpp]: Done.

File: physics/rigid_motion_body.hpp (hunk 3 above)
Comment (eggrobin) [inline physics/rigid_motion_body.hpp]: Done.

File: physics/rigid_motion_test.cpp (hunk 4 above)
Comment (eggrobin) [inline physics/rigid_motion_test.cpp]: Done.

File: physics/rigid_motion_test.cpp (hunk 5 above)
Comment (eggrobin) [inline physics/rigid_motion_test.cpp]: Done.

File: physics/rigid_motion_test.cpp (hunk 6 above)
Comment (eggrobin) [inline physics/rigid_motion_test.cpp]: Done.

**Pull request merged**